projects/vdk-core/tests/vdk/internal/builtin_plugins/ingestion/test_ingester_utils.py

```diff
@@ -4,7 +4,7 @@
 from decimal import Decimal
 
 from pytest import raises
-from vdk.internal.builtin_plugins.ingestion.ingester_utils import DecimalJsonEncoder
+from vdk.internal.builtin_plugins.ingestion.ingester_utils import IngesterJsonEncoder
 
 
 def test_decimal_json_encoder():
```

Comment:
We   need a new method test_timestamp_json_encoder 

Or rename the method payload_with_decimal_and_timestamps